LibVLCSharp/Shared/MediaPlayer.cs

```diff
@@ -1361,21 +1335,26 @@ public string AspectRatio
 
         public int Spu => Native.LibVLCVideoGetSpu(NativeReference);
 
-        public bool SetSpu(int spu)
-        {
-            return Native.LibVLCVideoSetSpu(NativeReference, spu) == 0;
-        }
+        /// <summary>
+        /// Set Spu (subtitle)
+        /// </summary>
+        /// <param name="spu">Video subtitle track to select (id from track description)</param>
+        /// <returns>true on success, false otherwise</returns>
+        public bool SetSpu(int spu) => Native.LibVLCVideoSetSpu(NativeReference, spu) == 0;
```

Comment:
Why do you have a property with only a getter, and a `Set...()` method? I'd prefer to have `Get...()` methods too for symmetry (and also, I'm not really a fan of getters that allocates memory).

Reply:
Just to be clear, this is outside the scope of this PR FYI as this API was already like this and this PR is not currently changing it.

> Why do you have a property with only a getter, and a Set...() method? 

Because the setter takes an argument, so it cannot be a traditional C# `get; set;`.

> I'd prefer to have Get...() methods too for symmetry

I.. don't :D and that'd make me Java-ify all the API.

> I'm not really a fan of getters that allocates memory

Well... it releases that memory as well to be fair, and looking into `MediaPlayer` properties at runtime with the debugger seems to be fine.

Reply:
OK, no strong opinions here.